packages/owl-bot/src/owl-bot.ts

```diff
@@ -147,13 +147,34 @@ export function OwlBot(
     }
   });
 
+  // Ensure up-to-date configuration is stored on merge
+  app.on('pull_request.merged', async context => {
+    const configStore = new FirestoreConfigsStore(db!);
+    const installationId = context.payload.installation?.id;
+    const org = context.payload.organization?.login;
+
+    logger.info('Scanning GitHub for configs (via `pull_request.merged`)');
+
+    if (!installationId || !org) {
+      logger.warn(`Missing install id (${installationId}) or org (${org})`);
+      return;
+    }
+
+    await scanGithubForConfigs(
```

Comment:
This will scan about 500 repos every time a pull request is merged.  I'm worried we'll use up our github quota.

Instead, call `refreshConfigs()` for the one repo into which the pull request was merged.

Reply:
Fixed